FXSkins/src/main/resources/com/pixelduke/control/skin/fx-skins.css

```diff
@@ -88,6 +88,10 @@ TextField > .right-button:pressed > .right-button-graphic {
 .password-field {
     -right-button-visible: true;
 
+    -fx-padding: 0.3333333em 0.5833333em 0.3333333em 0.5833333em;
```

Comment:
We already apply padding to the (normal) textfield (line 57). But we apply the one with the bigger right padding all the time as the right button is always visible.

Reply:
For consistency sake should we have the same right padding? The same right padding for textfield and passwordfield?

Reply:
```css
TextField {
    ...
    /*
    This is the same padding as the .password-field:focused
    We always need to apply it because the X-Button is always visible when there is text.
    */
    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em; 
}

.password-field {
    ...
    /*
    We remove the right padding when the password-field is not focused because the button is no longer visible and the text can extend to the entire width of the password-field.
    */
    -fx-padding: 0.3333333em 0.5833333em 0.3333333em 0.5833333em;
}

.password-field:focused {
    /*
    This is the same padding as the TextField
    We always need to apply it because the button is visible when the password-field is focused.
    */
    -fx-padding: 0.3333333em 3em 0.3333333em 0.5833333em;
}
```

Reply:
Ops sorry, wrong comment (sleeped very little today 😖). 
What I meant to say is whether we should also remove the right padding when the Textfield is not focused as we do for the passwordfield?
This is what happens if I add only "x" to the upper textfield and then bring the focus to another control (the bottom textfield):
![image](https://user-images.githubusercontent.com/2707487/117055567-34fe5880-ad13-11eb-8005-bf5b83b73c76.png)

The last x appears cut. So text can get cut right now if we fill the textfield and then put the focus on another control.